Client/Frontend/Browser/Browser.swift

```diff
@@ -11,8 +11,14 @@ protocol BrowserHelper {
     func userContentController(userContentController: WKUserContentController, didReceiveScriptMessage message: WKScriptMessage)
 }
 
+protocol BrowserDelegate {
+    func snackbarAdded(bar: SnackBar)
```

Comment:
I like snacking but I have no idea what a Snackbar is. Do we have a better name that is more informative?